benchmark/worker/echo.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const common = require('../common.js');
+const path = require('path');
+const bench = common.createBenchmark(main, {
+  workers: [1],
+  payload: ['string', 'object'],
+  sendsPerBroadcast: [1, 10],
+  n: [1e5]
+}, { flags: ['--experimental-worker'] });
+
+const workerPath = path.resolve(__dirname, '..', 'fixtures', 'echo.worker.js');
+
+function main(conf) {
+  const { Worker } = require('worker');
```

Comment:
BTW I think worker is not a suitable name since there's a mode called cluster-worker. I think people will be confused about whether worker is a sub-thread or sub-process.